redis/Redis.php

```diff
@@ -1508,7 +1508,7 @@ public function sContains($key, $value)
      * @param string $key
      *
      * @return int the cardinality of the set identified by key, 0 if the set doesn't exist.
-     *
+     * 
```

Comment:
Would you please, get rid of this spaces at the end of empty string?

Reply:
ok, fixed